sdk/Sdk/Targets/Microsoft.Azure.Functions.Worker.Sdk.targets

```diff
@@ -36,7 +41,16 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
   <UsingTask TaskName="ZipDeployTask"
             AssemblyFile="$(_FunctionsTaskAssemblyFullPath)"/>
 
-  <Target Name="_FunctionsPostBuild" AfterTargets="Build">
+    <Target Name="_FunctionsPreBuild" BeforeTargets="Build">
+        <Message Condition="'$(_AzureFunctionsNotSet)' == 'true'" Importance="high" Text="AzureFunctionsVersion not configured in the project. Setting AzureFunctionsVersion to v3"/>
+        <Error Condition="$(AzureFunctionsVersion.StartsWith('v1',StringComparison.OrdinalIgnoreCase))" Text="AzureFunctionVersion is set to an incompatible version, Please set it to v4"/>
```

Comment:
Should it be AzureFunctionsVersion instead of AzureFunctionVersion? (all error messages)

Reply:
`AzureFunctionsVersion` is the property name in the csproj file. Wanted to match the property name in the error message.